### PR TITLE
DDF-3058 Removed the obr repository

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Removes the obr repository
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@coyotesqrl @lessarderic 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Build](https://github.com/orgs/codice/teams/build)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by AzGoalie** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -631,6 +631,13 @@
                                 </execution>
                             </executions>
                         </plugin>
+                        <plugin>
+                            <groupId>org.apache.felix</groupId>
+                            <artifactId>maven-bundle-plugin</artifactId>
+                            <configuration>
+                                <obrRepository>NONE</obrRepository>
```
Comment (stustison) [inline pom.xml]: Can't we add this to the bundle plugin config that already exists further up around line 300?

**Comment by lessarderic** [comment]
❓ Wonder if we should add a comment to say _why_ that property is being set to `NONE`...

**Comment by stustison** [comment]
built manually and the build passed

**Pull request merged**